user_interface/EbikeUIController/EbikeUIController.ino

```diff
@@ -71,15 +81,15 @@ void timerISR() {
   if (currentIncreaseThrottleState != previousIncreaseThrottleState &&
       currentIncreaseThrottleState == HIGH &&
       currentTime - lastThrottleTime > 25) { // Debounce time of 25ms
-    handleThrottleInput(1, throttle, activatedRegen);  // Increase throttle request
+    handleThrottleInput(INCREASE, throttle, activatedRegen, DIGITAL);  // Increase throttle request
     lastThrottleTime = currentTime; // Update last throttle event time
   }
 
   // Check for throttle decrease with debouncing
   if (currentDecreaseThrottleState != previousDecreaseThrottleState &&
       currentDecreaseThrottleState == HIGH &&
       currentTime - lastThrottleTime > 25) { // Debounce time of 25ms
-    handleThrottleInput(2, throttle, activatedRegen); // Decrease throttle request
+    handleThrottleInput(DECREASE, throttle, activatedRegen, DIGITAL); // Decrease throttle request
```

Comment:
The added enums and switch inside this function have increased readability here, but I think we can do better by making a single function that handles all the debouncing and returns a throttle request type. Our code would then look something like:

`
throttleRequestType = debounceControls(<state variables for debouncing>); 
handleThrottleInput(throttleRequestType, throttle, activatedRegen, regenMethod);
`